packages/runner/lib/app.ts

```diff
@@ -40,9 +40,8 @@ try {
     });
 
     process.on('unhandledRejection', (reason) => {
-        logger.error(`${id} Received unhandledRejection...`, reason);
-        process.exitCode = 1;
-        close();
+        logger.error(`${id} Received uncaughtException...`, reason);
+        // not closing on purpose
```

Comment:
doing the same thing as for `uncaughtException` below

Reply:
The problem is that misusing promises in scripts bypasses the exec try/catch and was killing the entire runner process. I would love to handle those promise rejections but I am not sure how at the moment